packages/TorneloScoresheet/src/pages/ResultDisplay/SendEmail.tsx

```diff
@@ -0,0 +1,24 @@
+import qs from 'qs';
+import { Linking } from 'react-native';
+
+export async function sendEmail(to: string, subject: string, body: string) {
+  let url = `mailto:${to}`;
+
+  // Create email link query
+  const query = qs.stringify({
+    subject: subject,
+    body: body,
+  });
+
+  if (query.length) {
+    url += `?${query}`;
+  }
+  // check if we can use this link
+  const canOpen = await Linking.canOpenURL(url);
+
+  if (!canOpen) {
+    throw new Error('Provided URL can not be handled');
+  }
+
+  return Linking.openURL(url);
+}
```

Comment:
I feel like this should probably be a function that lives in the `util` directory,
maybe in a new file called `emailUtils` ?
I mean we can leave it here but its better to leave only UI stuff in the pages directory